routes/web.php

```diff
@@ -9,6 +9,16 @@
     return Inertia::render('LandingPage');
 })->name('landing');
 
+Route::get('login', [AuthenticatedSessionController::class, 'create'])
```

Comment:
```suggestion
Route::get('/login', [AuthenticatedSessionController::class, 'create'])
```
shouldnt this be `/login`?

Reply:
yes, it should be `/login` instead.